content/help/ops/setup/webhook/index.md

```diff
@@ -56,3 +56,8 @@ webhooks and dependent features are not functioning properly.
    api-server operation (see related issues
    [here](https://github.com/kubernetes/kubernetes/pull/58698#discussion_r163879443)
    and [here](https://github.com/kubernetes/kubeadm/issues/666) for more information).
+
```

Comment:
Can you merge this new section with the previous numbered point ("verify the Kubernetes api-server has network connectivity ..."? Both reference incorrect proxy environment settings (`no_proxy` and `http_proxy`) when using kubeadm.